src/test/unit/ntypes/ArrayTest.cpp

```diff
@@ -588,23 +588,11 @@ TEST_F(ArrayTest, testArrayBasefunctions) {
     //std::cerr << "ArrayTest:testArrayBasefunctions a=" << a << std::endl;
     nCols = testdata.size();
 
-    // getMaxElementsCount, getCount, setCount, copy
-    if (testCase->second.dataType != NTA_BasicType_SDR) {
```

Comment:
TODO is this temporary? So currently we cannot serialize a SDR?

Reply:
> TODO is this temporary? 

Yes, it is temporary.  I did not have SDR serialization completely added yet.   That is coming.